patroni/postgresql/bootstrap.py

```diff
@@ -400,9 +400,9 @@ def post_bootstrap(self, config: Dict[str, Any], task: CriticalTask) -> Optional
 END;$$""".format(f, quote_ident(rewind['username'], postgresql.connection()))
                         postgresql.query(sql)
 
-                for name, value in (config.get('users') or {}).items():
-                    if all(name != a.get('username') for a in (superuser, replication, rewind)):
-                        self.create_or_update_role(name, value.get('password'), value.get('options', []))
+                if config.get('users'):
+                    logger.error('user creation is not be supported after v4.0.0')
+                    raise Exception('user creation is not be supported after v4.0.0')
```

Comment:
I don't think we need to raise an exception.
```suggestion
```

Reply:
can an exception remind the user of Patroni that their Postgres users are not created?

Reply:
This exception may interrupt a log running custom bootstrap (for example restoring 10TB database from the backup).

Reply:
make sense